package.json

```diff
@@ -26,6 +26,9 @@
   "homepage": "https://github.com/skroutz/skroutz-analytics-woocommerce#readme",
   "devDependencies": {
     "grunt": "^1.0.1",
-    "grunt-wp-i18n": "^0.5.4"
+    "grunt-contrib-clean": "^1.0.0",
+    "grunt-contrib-copy": "^1.0.0",
+    "grunt-wp-i18n": "^0.5.4",
+    "grunt-wp-plugin": "skroutz/grunt-wp-plugin#v1.1.0-skroutz"
```

Comment:
[MINOR] I would prefer to use this one because does not depend on git but there is not need to change it 👍 
```bash
npm install --save-dev https://github.com/skroutz/grunt-wp-plugin/tarball/v1.1.0-skroutz
```